common/coap-server/src/main/java/org/thingsboard/server/coapserver/TbCoapDtlsSettings.java

```diff
@@ -108,6 +117,15 @@ public DtlsConnectorConfig dtlsConnectorConfig(Configuration configuration) thro
                 configBuilder.set(DTLS_CONNECTION_ID_NODE_ID, null);
             }
         }
+        if (maxTransmissionUnit != null) {
```

Comment:
It can never be null, because we specify the default value. Check if it is `> 0`. Same for maxFragmentLength

Reply:
Ok